src/widgets/ot_drawer.eliom

```diff
@@ -152,10 +155,12 @@ let%shared drawer
 
   let _ = [%client (
     let%lwt () = Ot_nodeready.nodeready (To_dom.of_element ~%d) in
-    (Eliom_client.onunload @@ fun () ->
-    html_ManipClass_remove "ot-drawer-opening";
-    html_ManipClass_remove "ot-drawer-open";
-    html_ManipClass_remove "ot-drawer-closing");
+    Eliom_client.Page_status.oninactive (fun () ->
```

Comment:
Do we still need the `Ot_nodeready.nodeready` right above?

Reply:
Removed.